tests/main/views/test_marketplace.py

```diff
@@ -1298,6 +1298,47 @@ def test_catalogue_of_briefs_page_filtered_all_options_selected(self):
         ss_elem = document.xpath("//p[@class='search-summary']")[0]
         assert self._normalize_whitespace(self._squashed_element_text(ss_elem)) == "6 results"
 
+    def test_opportunity_data_header_visible_on_catalogue_page(self):
```

Comment:
These four tests are really only checking one thing i.e. that the `_opportunity_data` partial has rendered OK.  So I'd be tempted to have these as a single test with four assertions about the different bits of content we care about.

But as things stand there is no way that one test could fail and the other pass - they either all fail or all pass, which to me makes it feel like a test if just one thing.

I don't really think we need to assert every bit of content - we should be able to tweak wording in the template and not have unit tests fail.  I'm fine with all the assertions if you like, but I'd say the one thing we rally care about testing is that the link is there and points to the right place.

Reply:
I had one test initially, but then I refactored it into four tests, because when you have one assertion per test it is much easier to see what broke/changed once something does break or change. And I disagree about all having to either pass or fail. When I was fiddling with various parts of content and html elements yesterday following reviews from content/design and product manager, some tests broke but some didn't, confirming exactly what has changed and what remained the same. I feel that this was very helpful and would be diminished if I bunched them all together.

Reply:
It's just as easy to see what broke when you have multiple assertions in a test - you can add an error message for each assertion, e.g. 
```
assert "this" in some_list, "This was not in the list"
assert x > 20, "x was less than 20"
assert expected_desc in description, "Could not find expected description in the page"
```

And then you get a nice message telling you exactly what has gone wrong.  And this will save precious milliseconds of test run time loading exactly the same page four times over :)

But fine if you want to leave it as four.

Reply:
Looking at the file history I think it is what I originally thought - the DOS list of suppliers csv downloads from the requirements start pages.  I think..